accounting_firm/demo/mail_activity.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="mail_activity_2" model="mail.activity">
+        <field name="res_model_id" ref="account.model_account_journal" />
+        <field name="res_id" ref="account.1_general" />
```

Comment:
Are we sure this will work in all cases (like existing databases)?

Reply:
Yes, as coming from native module.

Reply:
1 is related to company with id 1 if I'm not mistaken. Should be ok